user/pages/04.Reference/10.get-quota-info/docs.en.md

```diff
@@ -84,17 +94,12 @@ The response contains the quota information in JSON format. Example for a respon
     "volume.space_gb": 1000,
     "volume.volumes": 1024
   },
-  "dbl": {
+  "fes": {
     "compute.cores": 60,
     "compute.instances": 60,
     "compute.ram_mb": 245760,
     "dns.zones": 10,
     "network.floatingips": 50,
```

Comment:
where are the new octavia loadbalancer limits in this example?

Reply:
added them